tester_app/REACT_ON_RAILS.md

```diff
@@ -0,0 +1,20 @@
+The `react_on_rails` gem has been installed. You can view the documentation online at
+[React on Rails](https://github.com/shakacode/react_on_rails).
+
+Also, check out the [example application](https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/server.js)
+for a live example and code.
+
+
+The "Hello World" example has been installed.
+
+The view is located at `app/views/hello_world/index.html.erb`
+The controller is located at `app/controllers/hello_world_controller.rb`
+
+See [the documentation](https://github.com/shakacode/react_on_rails) for how to build your bundles and
+install your packages. Then you can view the example as follows:
+
+-"Hello World" example: [localhost:3000/hello_world](localhost:3000/hello_world)
+-"Hello World" example via Webpack Development Server with Hot Module Reload: `localhost:4000`
```

Comment:
![2015-11-01_19-50-06](https://cloud.githubusercontent.com/assets/1118459/10874004/de21ee50-80d1-11e5-8876-974c3656cef4.png)


Provide command to start (foreman -f etc.)